ci_support/ensure_sundials_installed.sh

```diff
@@ -2,7 +2,7 @@
 
 SUNDIALS_DEFAULT_VERSION='2.7.0'
 
-export SUNDIALS_DIR=$HOME/sundials/"${SUNDIALS_VERSION:-$SUNDIALS_DEFAULT_VERSION}"
+export SUNDIALS_DIR=$HOME/sundials/"${SUNDIALS_VERSION:-$SUNDIALS_DEFAULT_VERSION}"/"${SUNDIALS_PRECISION:-'double'}"
```

Comment:
If I test this on my PC, I see eg 

    Installing: /home/benny/sundials/3.1.0/'double'/lib/libsundials_kinsol.a

Would without the apostrophe 'double' not be better?

Reply:
Yeah, apparently I double quoted, fixed.